hack/test-cmd.sh

```diff
@@ -74,20 +74,21 @@ OPENSHIFT_ON_PANIC=crash openshift start --master="${API_SCHEME}://${API_HOST}:$
 OS_PID=$!
 
 if [[ "${API_SCHEME}" == "https" ]]; then
-    export CURL_CA_BUNDLE="${CERT_DIR}/admin/root.crt"
+    export CURL_CA_BUNDLE="${CERT_DIR}/system:admin/root.crt"
```

Comment:
If you change directories, we need to make sure we update ALL the places that reference it (docs, howtos, scripts to copy things to minion vagrant images, etc). Probably easier to leave the directory alone and just add the provider name in the cert itself.